float32 bus_voltage
float32 bus_current
float32 fet_temperature
float32 motor_temperature
ActiveErrors active_errors
ActiveErrors disarm_reason
